lib/spack/docs/build_systems/intelpackage.rst

```diff
@@ -12,8 +12,9 @@ This build system is a work-in-progress. See
 https://github.com/spack/spack/pull/4300 and
 https://github.com/spack/spack/pull/7469 for more information.
 
+************
 Introduction
-~~~~~~~~~~~~
+************
```

Comment:
Right. I'm still new to .rst docs.

Since I use different section levels: Can I use [*numbered* sections](http://docutils.sourceforge.net/docs/ref/rst/directives.html#automatic-section-numbering) and [a TOC](http://docutils.sourceforge.net/docs/ref/rst/directives.html#table-of-contents)?

Reply:
The main Build Systems will contain a TOC. I think it's overkill to add a third level of TOC in addition to that and the main one.